README.md

```diff
@@ -23,9 +23,19 @@
 
 ## Being a better company to work at
 
+- [ ] Setup the ML cluster environment
 - [ ] Regression Test, Benchmarks and Deploy platform (CD?)
+    - For which projects? Are any of ML included? - Vadim
 - [ ] Improve Logs, CLI, and other guides.
+    - Logs for Go or for Python too? CLI guide - do we have one?
 - [ ] Make it easy to update and maintain PGA.
+- [ ] Conduct a happiness survey and react to the unveiled issues
+- [ ] Improve the formal process of speaking at a conference
+    - We have nothing currently. Even a simple checklist would save us much time and nerves, especially to poor Esther who really struggled with my conferencing season. It starts with notifying her once the approval registered in src-d/conferences and ends with submitting the expenses report. It also includes what to check in the booking confirmation because I had to buy tickets at the airport one hour before the flight two times already and that sucked very much. It also includes the Visa knowledge database, e.g. British visas are tricky.
+- [ ] Increase the transparency of the events happening at source{d}
```

Comment:
This is not a OKR

Reply:
This was forwarded to Slack. Someone else (not engineering) will check if it's part of their OKRs.

Reply:
If it is not, we just silently drop it or what?

Reply:
This is something that you must to request to the devrel team, this is not a engineering task for sure

Reply:
Not sure if devrel or office or management but I get your point: not an engineering task

Reply:
@vmarkovtsev You can drop it silently for the PR, it's been circulated in the #okrs channel on Slack.

Reply:
Or shall it be suggested to move i.e in https://github.com/src-d/devrel/pull/67 ?

Reply:
Added to our OKRs @bzz @smola